### DISCUSSION TITLE
ftp timestamp off in yi-hack-MStar v0.4.7

### DISCUSSION THREAD

**Discussion by blfriedman** [roleoroleo/yi-hack-MStar#439] [General]
I have recently updated a Yi 1080p camera from yi-hack-MStar v0.4.3 to v0.4.7.  No configuration changes were associated with the upgrade.

The problem is that after the upgrade, I noticed that the directory names on my FTP server created by the camera are four hours into the future.

Info on system with ftp file dates off by +4 hours:
base version is: 2.1.0.0E_201809191630
model suffix is: y23

I also have another Yi 1080p camera I have not updated, still running v0.4.3 with the same base version and same model suffix.  It does not exhibit this problem.

Both cameras are setup with the same information for the ftp server, and both send all files to the ftp server and delete them locally.

The General Configuration for the Timezone on both cameras is: CST6CDT,M3.2.0,M11.1.0
this is the timezone used for Chicago, IL, USA.

There are no differences in the configuration of the cameras other than the identification information.

Both cameras have good wifi signal and are staying on the network - (I monitor them continuously, and neither has shown a problem).

Were there changes in the ftp server w.r.t. directory naming?

**Comment by blfriedman** [comment]
After a second reboot of the camera after the initial reboot upon installation of the new firmware , this issue seems to have cleared and the ftp directory names are correct.

**Comment by roleoroleo** [comment]
I was testing the reported issue and actually couldn't figure it out.
Because the name of the file (folder + file) written is the same as the one read.
So I'm glad the problem is gone.
